addons/website_event_questions/models/event_event.py

```diff
@@ -32,37 +33,28 @@ def _compute_question_ids(self):
 
         When synchronizing questions:
 
-          * lines that no answer are removed;
+          * lines with no registered answers are removed;
           * type lines are added;
         """
         if self._origin.question_ids:
             # lines to keep: those with already given answers
-            questions_tokeep_ids = self.env['event.registration.answer'].search(
-                [('question_id', 'in', self._origin.question_ids.ids)]
-            ).question_id.ids
+            questions_tokeep = self.env['event.registration.answer'].search([
+                ('question_id', 'in', self._origin.question_ids.ids),
+            ]).question_id
         else:
-            questions_tokeep_ids = []
+            questions_tokeep = None
```

Comment:
Why changing this in stable ? Seems it does the same as before, just using recordserts instead of ids... this kind of code improvement belongs to master, unless you fixed something else ?